server/lib/transforms/link-analytics.js

```diff
@@ -1,13 +1,20 @@
 'use strict';
 
+const sanitise = text => text.replace(/[^\w ]/g, '');
```

Comment:
How essential is it for Spoor to get accurate link text? Would replacing with entities be better?